src/components/AssetForm.js

```diff
@@ -191,6 +192,8 @@ class AssetForm extends React.Component {
       this.props.formStore.fileControls.forEach(control => tabs.push([control.name, control.name]));
     }
 
+    tabs.push(["Searchables", "SEARCHABLES"]);
```

Comment:
We need to know when searchables should be shown. It should not be shown by default. This might be a new spec attribute unless we can determine any easier discriminating factors.

Reply:
Marc says he doesn't have a strong preference either way (show/hide). He says we only need it for search right now.